### ISSUE TITLE
Calcuttan Biryani

### ISSUE THREAD

**Issue by bloom-git** [kyscg/chickenBiryani#5]
Bro Calcuttan Biryani is best. It has  aloos and chickens and an egg. All awards goes to this.

**Comment by kyscg** [comment]
good idea, let this issue mark the request of good recipes for Calcuttan biriyani

Contributions welcome.
